auth/test/TestHashingWrappers.m

```diff
@@ -8,6 +8,30 @@ function test_hello_world(testCase)
             test_output = HashingWrappers.sha_256(input);
             testCase.assertEqual(correct_output, test_output);
         end
+        
+        function test_HMAC(testCase)
+            input = matlab.net.base64decode(matlab.net.base64encode("Hello World"));
+            key = matlab.net.base64decode(matlab.net.base64encode("ls"));
+            correct_output = uint8(sscanf('9e2b27474665d773b5009acab27a27a9167248cd71b9a783fc00660b487cbe6c', '%2x'));
+            test_output = HashingWrappers.hmac_sha_256(input, key);
+            testCase.assertEqual(correct_output, test_output);
+        end
+        
+        function test_authenticator_ECDSA(testCase)
```

Comment:
These tests pertain to the AuthenticatorECDSA, so put them into a TestAuthenicatorECDSA class.

Add these tests:
false = verify(...) upon
message is modified - you have this one
signature is modified
a new different Authenticator attempts to verify correct (message, signature) because it has the wrong public key

Reply:
add this test:

multiple signatures and verifications...in practice, the same object will sign and verify lots of signatures